### PR TITLE
Define ScmInfo

### PR DESCRIPTION
**Pull request by patriknw**
* it was previously added by sbt-git, but we removed that plugin

### CODE REVIEW

**Comment by raboof** [comment]
(travis failure is probably #27815)

File: project/AkkaBuild.scala [hunk 1]
```diff
@@ -160,7 +160,7 @@ object AkkaBuild {
     licenses := Seq(("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0.html"))),
     homepage := Some(url("https://akka.io/")),
     description := "Akka is a toolkit for building highly concurrent, distributed, and resilient message-driven applications for Java and Scala.",
-
+    scmInfo := Some(ScmInfo(url("https://github.com/akka/akka"), "git@github.com:akka/akka.git")),
```
Comment (raboof) [inline project/AkkaBuild.scala]: I don't think I usually include `.git` - can't hurt though

**Review by raboof — APPROVED** [review]

File: project/AkkaBuild.scala (hunk 1 above)
Comment (patriknw) [reply project/AkkaBuild.scala]: I just copied from another place where I knew it worked.
It was previously (in M7):
```
    <scm>
        <url>https://github.com/akka/akka</url>
        <connection>scm:git:https://github.com/akka/akka.git</connection>
        <developerConnection>scm:git:git@github.com:akka/akka.git</developerConnection>
    </scm>
```

We can define it to be the same as before.

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by raboof** [comment]
(travis error was #27815)

**Comment by patriknw** [comment]
We shouldn't care about defining it in the same ways as before?
https://github.com/akka/akka/pull/27817#discussion_r329362144
I don't know how this information is used.

**Comment by raboof** [comment]
Ah, good one, added #27833

**Pull request merged**